docs/install.md

```diff
@@ -138,6 +138,8 @@ best suits your business needs. For example:
  - In some environments, creating a persistent volume could be slower than transferring files to/from a cloud storage bucket.
  - If the cluster is running in multiple zones, accessing a persistent volume could be unreliable.
 
+**Note:** To customize the names of the `ConfigMaps` for artifact persistence (e.g. to avoid collisions with other services), rename the `ConfigMap` and update the env value defined [controller.yaml](https://github.com/jerop/pipeline/blob/9389134907e7792a0b01d1a231d1b6b4441226b5/config/controller.yaml#L66-L75).
```

Comment:
It might be a good idea to point to `https://github.com/tektoncd/pipeline` rather than a personal repository.

Reply:
good catch, will update, thanks!